### PR TITLE
Fix logging

### PR DESCRIPTION
**Pull request by insacuri**
**What is the context of this PR?**

Link to Trello card

Describe what you have changed and why, link to other PRs or Issues as appropriate.

**How to review**

Any notes for the reviewer  (include screenshots if appropriate).

### CODE REVIEW

File: secure_message/resources/health.py [hunk 1]
```diff
@@ -43,7 +43,6 @@ def get():
         details = {'Name': current_app.config['NAME'],
                    'Version': current_app.config['VERSION'],
                    'SMS Log level': current_app.config['SMS_LOG_LEVEL'],
-                   'APP Log Level': current_app.config['APP_LOG_LEVEL'],
```
Comment (dcdarrell9) [inline secure_message/resources/health.py]: in test_health_monitor.py I think it's looking for this, which is causing the test to fail.

File: Makefile [hunk 2]
```diff
@@ -12,4 +12,5 @@ lint:
 	pipenv check ./secure_message ./tests
 
 test: lint
-	pipenv run python run_tests.py
+	pipenv run behave --format progress
+	export APP_SETTINGS=TestConfig && pipenv run python -m unittest && unset APP_SETTINGS
```
Comment (dcdarrell9) [inline Makefile]: I think we could probably change this to just `pipenv run pytest` I think the run_tests.py file is a bit bloated especially when this does the same thing?

Comment (insacuri) [reply Makefile]: Forgot pytest even existed... thanks ^^

File: secure_message/resources/health.py (hunk 1 above)
Comment (insacuri) [reply secure_message/resources/health.py]: Thanks for fixing that

File: Makefile [hunk 3]
```diff
@@ -12,4 +12,5 @@ lint:
 	pipenv check ./secure_message ./tests
 
 test: lint
-	pipenv run python run_tests.py
+	pipenv run behave --format progress
+	export APP_SETTINGS=TestConfig && pipenv run pytest && unset APP_SETTINGS
```
Comment (JamesGardiner) [inline Makefile]: I don't think exporting this here will have any effect on the environment in which pipenv is executing the pytest script from. You'll need to use a .env for that or run the export and unset inside the pipenv shell

File: README.md [hunk 4]
```diff
@@ -66,17 +66,16 @@ Ensure there is a postgres instance running on port 5432
 docker run -d -p 5432:5432 --name postgres -e POSTGRES_PASSWORD=postgres -e POSTGRES_DB=postgres -d postgres
 ```
 
-Run them using coverage
+Run the with make
```
Comment (dcdarrell9) [inline README.md]: 🎨

**Pull request merged**